ui/pages/settings/index.scss

```diff
@@ -296,6 +296,16 @@
       margin-top: 10px;
     }
 
+    &__dropdown {
+      height: auto;
+    }
+
+    &__title {
+      font-size: 14px;
+      font-weight: 500;
+      line-height: 22px;
+    }
```

Comment:
wondering if we should be using the component library typography variables here, and below for things like `font-size`, `letter-spacing` etc

Reply:
I see that we could do some refactoring and use `FontWeight` and `letterSpacing` for example; although for font-size not sure there is a dedicated typography prop that we can use? I pushed some updates related to `settings-page__content-item__description` and `settings-page__content-item__title` CSS 🙏